CONTRIBUTING.md

```diff
@@ -46,23 +47,23 @@ your interactions with the project and its participants.
 ## Observability Pack Guidelines
 
 We encourage all contributors to actively engage in the creation and maintenance of
-Observability Packs. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
+Quick starts. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
```

Comment:
based on the last thread i read (which was a minute ago) i _think_ we're going with lowercase `quickstarts` unless it's the beginning of a sentence, does that sound right to you?

Reply:
That is correct!

Reply:
I knew I forget this at least a dozen times before it sunk in. 🤣